tests don't compile on 1.34.0

While I can build the crate with `cargo +1.34.0 build` just fine, trying to test it on that version fails with this error:

```
error[E0658]: use of unstable library feature 'alloc': this library is unlikely to be stabilized in its current form or name (see issue #27783)
  --> /home/jonas/.cargo/registry/src/github.com-1ecc6299db9ec823/rand_core-0.6.2/src/lib.rs:45:27
   |
45 | #[cfg(feature = "alloc")] extern crate alloc;
   |                           ^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: Could not compile `rand_core`.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -33,3 +33,22 @@ jobs:
         rustup component add rustfmt-preview
         cargo fmt --all -- --check
       if: matrix.rust == 'stable'
+
+  test-msrv:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust: [1.34.0]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ matrix.rust }}
+        override: true
+    - name: Install libwebp
+      run: sudo apt-get update && sudo apt-get install libwebp-dev -y;
+    - name: Test MSRV
+      run: cargo build --all --features "libwebp-0_6"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,8 @@
+## Unreleased
+
+- Misc
+  - Test build on Rust 1.34.0 & update docs https://github.com/qnighy/libwebp-image-rs/pull/8
+
 ## 0.2.0
 
 - Changed

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ This is a bridge library for `libwebp` and `image`.
 
 ## Minimum Supported Rust Version (MSRV)
 
-Rust 1.34.0
+Rust 1.34.0 (You need >= 1.36.0 when developing the crate itself)
 
 ## Features
 

